command/config/validate_listener.go

```diff
@@ -0,0 +1,7 @@
+package config
```

Comment:
This will need to match the one on ENT exactly, so go ahead and add the build tag check here. One other way to do this in dev is to get the OSS side approved and merged, then merge oss->ent to pick up what you need ENT side, then get that one reviewed.  In this case it's fine just to have the file be identical